src/app_handle.rs

```diff
@@ -294,6 +294,43 @@ impl ApplicationHandle {
             if let Some(window_icon) = config.window_icon {
                 window_builder = window_builder.with_window_icon(Some(window_icon));
             }
+            #[cfg(target_os = "macos")]
+            if let Some(mac) = config.mac_os_config {
+                use floem_winit::platform::macos::WindowBuilderExtMacOS;
+                if let Some(val) = mac.movable_by_window_background {
+                    window_builder = window_builder.with_movable_by_window_background(val);
```

Comment:
Is there an option to make this something like `window_builder.opt_change_movable_by_window_background(&mac.movable_by_window_background);`? Not in this PR if there is need to minimize diff, but generally.

Reply:
I don't really understand what you're asking here.

You want to change the name of something? The character case? Or the functionality?

Reply:
Okay, seeing this on the computer I think I see what you mean.  `WindowBuilder` is part of [winit](https://github.com/rust-windowing/winit.git) - it is just used (and exported) by Floem.  The method name you want to change is not part of this project.